### PR TITLE
Dev Upgrade after main hot fix

### PR DESCRIPTION
**Pull request by fboucquez**
- main merge
- moved dev version to 1.1.0
- fixed unit tests

### CODE REVIEW

File: test/reports/bootstrap-voting/api-node-0-config.csv [hunk 1]
```diff
@@ -431,4 +434,4 @@ seedDirectory; ./seed
 certificateDirectory; ./cert
 dataDirectory; ./data
 pluginsDirectory; /usr/catapult/lib
-votingKeysDirectory; ./votingkeys
+votingKeysDirectory; ./votingkeys
```
Comment (yilmazbahadir) [inline test/reports/bootstrap-voting/api-node-0-config.csv:437]: no newline?

File: test/reports/bootstrap-voting/peer-node-0-config.csv [hunk 2]
```diff
@@ -441,4 +444,4 @@ seedDirectory; ./seed
 certificateDirectory; ./cert
 dataDirectory; ./data
 pluginsDirectory; /usr/catapult/lib
-votingKeysDirectory; ./votingkeys
+votingKeysDirectory; ./votingkeys
```
Comment (yilmazbahadir) [inline test/reports/bootstrap-voting/peer-node-0-config.csv:447]: newline?

File: test/reports/bootstrap-voting/peer-node-1-config.csv [hunk 3]
```diff
@@ -441,4 +444,4 @@ seedDirectory; ./seed
 certificateDirectory; ./cert
 dataDirectory; ./data
 pluginsDirectory; /usr/catapult/lib
-votingKeysDirectory; ./votingkeys
+votingKeysDirectory; ./votingkeys
```
Comment (yilmazbahadir) [inline test/reports/bootstrap-voting/peer-node-1-config.csv:447]: newline?

File: test/reports/bootstrap-voting/rest-gateway-0-rest.json [hunk 4]
```diff
@@ -91,4 +91,4 @@
     "deploymentToolVersion": "abc",
     "lastUpdatedDate": "2021-05-23"
   }
-}
+}
```
Comment (yilmazbahadir) [inline test/reports/bootstrap-voting/rest-gateway-0-rest.json:94]: newline?

Comment (fboucquez) [reply test/reports/bootstrap-voting/rest-gateway-0-rest.json:94]: Thanks, added the new lines back after lost in a geneation

**Review by yilmazbahadir — APPROVED** [review]

**Pull request merged**